### PR TITLE
Added QR Code Generator

### LINKED ISSUE #10: QR Code Generator
### Describe the new script you want
User will input a link and the script returns a QR code of that link

### Describe the solution you'd like us to follow
A python script is preferable

## Do you want to work on this?
- [ ] yes
- [x] no

### Additional context
Add any other context or screenshots about the feature request here. If not applicable remove this section.

### PR DESCRIPTION
**Pull request by danger-ahead**
# Description

Added a QR Code Generator that generates a QR code from the link provided by the user. Also it runs a validation check (before generating QR Code) to check if the inputted URL is valid. The QR Code generated is saved as both `.png` and `.svg` in the same directory.

Fixes #10 

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)
- [x] New Script (non-breaking change which adds a new script to the repository)

## Has This Been Tested?

- [x] I confirm that I have tested my script locally and verified that it works as intended.

## Screenshot (if applicable)
image

## Checklist:

- [x] I have gone through the contributing guidelines before making this Pull Request.
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

**Comment by OBITORASU** [comment]
Also lets add an optional flag which lets the user choose the save directory location as well.

**Comment by danger-ahead** [comment]
> Also lets add an optional flag which lets the user choose the save directory location as well. Also you forgot to edit the main README to add your script.

@OBITORASU have a look.

File: scripts/QR Code Generator/qr_code_gen.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Importing the necessary packages
+import pyqrcode
+import png
+from argparse import ArgumentParser
+import validators
+import os
+
+# Constructing an Argument Parser
+ap = ArgumentParser()
+ap.add_argument("-l", "--link", required=True, help="Path to the image folder")
```
Comment (OBITORASU) [inline scripts/QR Code Generator/qr_code_gen.py:10]: Change the help description I think it should be concerning the link instead of image dir.

**Review by OBITORASU — CHANGES_REQUESTED** [review]

**Comment by OBITORASU** [comment]
Last request modify the main readme and add your project to the list.

**Comment by OBITORASU** [comment]
LGTM Thanks!

**Review by OBITORASU — APPROVED** [review]

**Pull request merged**